addons/sourcemod/scripting/ttt/ttt.sp

```diff
@@ -1170,6 +1170,9 @@ stock void TeamInitialize(int client)
 		if (GetPlayerWeaponSlot(client, CS_SLOT_PRIMARY) == -1)
 		{
 			GivePlayerItem(client, g_iConfig[sdefaultPriD]);
+		} else {
```

Comment:
There should be a cvar to control this behaviour.

Reply:
Add if you want, I believe it's more than sufficient as is.

Reply:
or we remove the weapon (as option to config (0 - Nothing, 1 - Drop weapon or 2 - Delete weapon))

Reply:
or we remove the weapon (as option to config (0 - Nothing, 1 - Drop weapon or 2 - Delete weapon))